resources/js/components/StackedChart.vue

```diff
@@ -161,7 +161,7 @@
 
         if(this.chartTooltips !== undefined){
           this.options.plugins.tooltip = this.chartTooltips;
-          const tooltiplist = ["custom", "itemSort", "filter"];
+          const tooltiplist = ["external", "itemSort", "filter"];
```

Comment:
is it only applicable for stackedChart? I think it is also applicable to another chart? cmiiw.

If yes, can you help to update another from custom to another?